jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/form/BootstrapValidationTest.java

```diff
@@ -0,0 +1,145 @@
+        };
+    }
+
+    @Test(description = "Test validation rules with negative data", dataProvider = "negativeData")
+    public void simpleValidationNegativeTest(SimpleContact entity) {
+
+        FormValidationForm form = formPage.formValidationSection.form();
+        formPage.formValidationSection.switchToBrowserValidation();
+
+        form.fill(entity);
+        Assert.assertFalse(form.isValid());
+
+        form.reset();
+
+        formPage.formValidationSection.switchToCustomValidation();
+
+        form.fill(entity);
+        Assert.assertFalse(form.isValid());
+
+        form.reset();
+    }
+
+    @Test
+    public void bootstrapValidationTest() {
+
+        String name = "ValidName";
+        String email = "InvalidEmail";
+        String phone = "InvalidPhone";
+
+        SimpleContact entity = new SimpleContact(name, email, phone);
+
+        FormValidationForm form = formPage.formValidationSection.form();
+        formPage.formValidationSection.switchToCustomValidation();
+
+        form.fill(entity);
+        form.submit();
+
+        Map<String, String> validFeedback = form.getValidFeedback();
+
+        MatcherAssert.assertThat("Number of valid feedbacks not equals 1", validFeedback.size() == 1);
+        MatcherAssert.assertThat(validFeedback.keySet(), Matchers.hasItems("Name"));
+        MatcherAssert.assertThat(validFeedback.values(), Matchers.hasItem("Hi, " + name + "!"));
+
+        Map<String, String> invalidFeedback = form.getInvalidFeedback();
+        MatcherAssert.assertThat("Number of invalid feedbacks not equals 2", invalidFeedback.size() == 2);
+        MatcherAssert.assertThat(invalidFeedback.keySet(), Matchers.hasItems("Email", "Phone"));
+        MatcherAssert.assertThat(invalidFeedback.values(), Matchers.hasItems("Enter valid email!", "It doesn't look like a valid phone number"));
+
+        form.reset();
+
+    }
+
+    @Test
+    public void browserValidationTest() {
+        String name = "ValidName";
+        String email = "InvalidEmail";
+        String phone = "InvalidPhone";
+
+        SimpleContact entity = new SimpleContact(name, email, phone);
+
+        FormValidationForm form = formPage.formValidationSection.form();
+        formPage.formValidationSection.switchToCustomValidation();
+
+        form.fill(entity);
+        form.submit();
+
+        Map<String, String> validFeedback = form.getValidationMessages();
+
+        MatcherAssert.assertThat("", validFeedback.get("Email"),
+                Matchers.is("Please include an '@' in the email address. 'InvalidEmail' is missing an '@'."));
+        MatcherAssert.assertThat("", validFeedback.get("Phone"),
+                Matchers.is("Please match the requested format."));
+        MatcherAssert.assertThat("", validFeedback.get("Name"), Matchers.is(""));
+
+        form.reset();
+    }
+
+    @AfterMethod
+    public void reset() {
+        formPage.formValidationSection
+                .form()
```

Comment:
Please, rename this method to getForm() or sort of

Reply:
Done.